### ISSUE TITLE
Could IResponse export?

### ISSUE THREAD

**Issue by idy** [node-influx/node-influx#243]
I use `InfluxDB.queryRaw` to select some data, and I want to prevent the magic tranformation. Indeed, I can give a `IResponse` type to the return data.

Could the `IResponse` interface export from `results.ts` ?

```typescript
let result:IResponse = await influx.queryRaw(`select * from sys_info`)
```

**Comment by connor4312** [comment]
Thanks for the report! Fixed in 6442686 and published in 5.0.2.

**Comment by idy** [comment]
Hi connor, Thanks for pr, but I want the `IResponse` to be exported, not the `IResults` ;)

**Comment by connor4312** [comment]
Whoops. Published in 5.0.3 ;)

**Comment by idy** [comment]
Thanks 👍
